code/controllers/subsystem/tgui.dm

```diff
@@ -286,6 +286,7 @@ SUBSYSTEM_DEF(tgui)
  */
 /datum/controller/subsystem/tgui/proc/on_open(datum/tgui/ui)
 	LAZYOR(ui.src_object.open_uis, ui)
+	ui.user.tgui_open_uis |= ui
```

Comment:
add a bubber edit comment
also, wont this mess with statpanel and chat? im sure this is already tracked somewhere

Reply:
what was also get duplicate of the same ui, will it close those with that operator?

Reply:
this was a misdelete caused by https://github.com/tgstation/tgstation/pull/76215
`tgui_open_uis` is used to track every ui currently open by an user while `all_uis` is used to track every ui currently open by every user
i have tested the code and this will not mess with either tgui chat or tgui statpanel

not sure what you meant by the second question, please rephrase

Reply:
fixed in https://github.com/tgstation/tgstation/pull/77101